modelling/src/main/java/org/axonframework/modelling/saga/AnnotatedSagaManager.java

```diff
@@ -175,6 +175,12 @@ public Builder<T> spanFactory(SpanFactory spanFactory) {
             return this;
         }
 
+        @Override
```

Comment:
Does the `@Deprecated` method bubble up through an override? If not, I think we should also deprecate the override of the `spanFactory(SpanFactory)` method.

Reply:
Yes it does! :)

Reply:
I thought it did, but it does not! :( I will add it for all instances